nautobot/extras/tests/test_api.py

```diff
@@ -1564,7 +1570,7 @@ def test_run_job_no_worker(self, mock_get_worker_count):
         response = self.client.post(url, data, format="json", **self.header)
         self.assertHttpStatus(response, status.HTTP_503_SERVICE_UNAVAILABLE)
         self.assertEqual(
-            response.data["detail"], "Unable to process request: No celery workers running on queue default."
+            response.data["detail"].title(), "Unable To Process Request: No Celery Workers Running On Queue Default."
```

Comment:
Why this change?

Reply:
It was failing in CI but not locally. Reverted it to see what happens

Reply:
Reverted and it is passing CI